script/bootstrap

```diff
@@ -18,9 +23,23 @@ fi
 ruby_version=`cat .ruby-version`
 
 echo "==> Installing Ruby: $ruby_version"
-ruby-install ruby $ruby_version --no-reinstall
-source /usr/local/opt/chruby/share/chruby/chruby.sh
-chruby $ruby_version
+if command_exists rbenv
+then
```

Comment:
Will you put these `then`s on the same line as the `if`/`elif`s please?

Reply:
done